kokoro/windows/build.bat

```diff
@@ -50,17 +51,17 @@ REM supports only ".pkg.tar.xz" package archives, it does not support
 REM the more recent ".pkg.tar.zst" packages archives.
 c:\tools\msys64\usr\bin\bash --login -c "pacman -R --noconfirm catgets libcatgets"
 REM Use an old version of patch known to work with the msys runtime
-REM version that comes on Kokoro.
-set REPO_MSYS2=https://sourceforge.net/projects/msys2/files/REPOS/MSYS2/x86_64
-set REPO_MINGW=https://sourceforge.net/projects/msys2/files/REPOS/MINGW/x86_64
```

Comment:
nit: looks like we could still keep the variables here and only replace the link .